test/e2e/manifests/values/base.yaml

```diff
@@ -65,6 +73,14 @@ storageClasses:
       allowVolumeExpansion: true
       additionalParameters:
         '{{ include "topolvm.pluginName" . }}/device-class': "thin"
+  - name: topolvm-provisioner-thin-btrfs
```

Comment:
This SC looks unused.

Reply:
Good catch! Oversight on my part, I forgot to add snapshot tests for btrfs before.

Reply:
Tests added